docs/debug/index.md

```diff
@@ -47,6 +48,49 @@ iteration    number    sum_so_far    avg_so_far    list_so_far
 6            60        210           35.0          [(10, 10.0), (30, 15.0), (60, 20.0), (100, 25.0), (150, 30.0), (210, 35.0)]
 ```
 
+### While Loop Example
+
+To use Accumulation Table with While Loops, you need to pass in the name of the loop variable when initializing the Table.
```

Comment:
Use the code font and class name for `AccumulationTable`. Again, don't capitalize "While Loops" or "table".